pkg/utils/utils_test.go

```diff
@@ -260,6 +261,32 @@ func TestCastToType_Ok(t *testing.T) {
 			value:    []byte{0xff},
 			expected: true,
 		},
+
+		// utf-8 string
+		{
+			typeName:       "t",
+			value:          []byte{},
+			expected:       "",
+			expectedLength: 0,
+		},
+		{
+			typeName:       "string",
+			value:          []byte(nil),
+			expected:       "",
+			expectedLength: 0,
+		},
+		{
+			typeName:       "utf8",
+			value:          []byte{0x31, 0x38, 0x30, 0x35, 0x32, 0x32, 0x30, 0x32, 0x31, 0x38, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+			expected:       "1805220218",
+			expectedLength: len("1805220218"),
+		},
+		{
+			typeName:       "t",
+			value:          []byte{0x34, 0x2e, 0x30, 0x2e, 0x36, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+			expected:       "4.0.6",
+			expectedLength: len("4.0.6"),
```

Comment:
doesn't look like `expectedLen` is tested for below

Reply:
Updated and added t16 because it looks like I need the length here: https://github.com/vapor-ware/vapor-charts/blob/18867b6eabd001024858fff046c01240da1d76e4/charts/egauge/templates/configmap.yaml#L36